mockatests/drivers/test-suites.h

```diff
@@ -46,6 +46,22 @@ static const struct CMUnitTest tapdisk_stats_tests[] = {
 	cmocka_unit_test(test_stats_realloc_buffer_edgecase)
 };
 
+void test_vbd_linked_list(void **state);
+void test_vbd_complete_td_request(void **state);
+void test_vbd_issue_request(void **stat);
+void test_vbd_complete_block_status_request(void **stat);
 
+static const struct CMUnitTest tapdisk_vbd_tests[] = {
+	cmocka_unit_test(test_vbd_linked_list),
+	cmocka_unit_test(test_vbd_issue_request),
+	cmocka_unit_test(test_vbd_complete_block_status_request)
+};
+
+/*void test_nbdserver_new_protocol_handshake(void **state);
```

Comment:
Not calling the new tests?

Reply:
Ahh yes, the i'm getting strange errors from cmocka.

I will attempt to fix them.